packages/cdk/lib/env/context-app-parameters.ts

```diff
@@ -151,4 +156,22 @@ export class ContextAppParameters {
       },
     };
   }
+
+  public getDefaultFilesystem(): string {
+    let defFilesystem: string;
+    switch (this.engineName) {
+      case "cromwell":
+        defFilesystem = "S3";
+        break;
+      case "nextflow":
+        defFilesystem = "S3";
+        break;
+      case "miniwdl":
+        defFilesystem = "EFS";
+        break;
+      default:
```

Comment:
Ask Elliot if he wants to add snakemake to this swith

Reply:
Yes, snakemake requires EFS so it would need to be here as well